styleguide.config.js

```diff
@@ -46,14 +46,22 @@ module.exports = {
           components: 'src/molecules/asides/**/*.js'
         },
         {
-          name: 'Headers',
-          components: () => [ 'src/molecules/PriceDiscountHeader/index.js' ]
-        },
-        {
           name: 'DataRow',
           components: 'src/molecules/DataRow/**/*.js'
         },
         {
+          name: 'FlashMessage',
+          components: 'src/molecules/FlashMessage/**/*.js'
+        },
+        {
+          name: 'FormFields',
+          components: 'src/molecules/formfields/**/*.js'
+        },
+        {
+          name: 'Headers',
+          components: () => [ 'src/molecules/PriceDiscountHeader/index.js' ]
```

Comment:
We shouldn't need to explicitly write these in every time. They should just get brought in by the "Other" category.